src/item_group.cpp

```diff
@@ -209,7 +209,8 @@ void Item_modifier::modify( item &new_item ) const
     }
 
     new_item.set_damage( rng( damage.first, damage.second ) );
-    double random_dirt = rng( dirt.first, dirt.second );
+    double random_dirt = static_cast<double>( rng( static_cast<int>( dirt.first ),
```

Comment:
you can use `rng_float` here

Reply:
I suspected that was the problem and did that and another change. Hopefully it will work now.

Reply:
Any idea what's still causing this? Will your RNG float fix it? 
![image](https://user-images.githubusercontent.com/45959491/65812934-3d3bfc80-e19c-11e9-94e6-b87d60b30465.png)

Reply:
That's because the variable initialization in the constructor has the wrong order as they are declared in the class definition.

Reply:
thanks.